migration-guide/business-central.adoc

```diff
@@ -0,0 +1,150 @@
+assets 'that are' created in Business Central so that they can be maintained by developers by using either {DEV_TOOLS} or {CANVAS} or both.
+== Prerequisites
+
+The following tools are required to migrate from Business Central to {DEV_TOOLS}. It is assumed that you have a current version of Business Central running to export 
+certain types of assets to their supported form in {PRODUCT_SHORT}:
+
+include::../partials/prerequisites.adoc[]
+
+== Business Automation Assets in Businss Central
+
+The following types of business automation assets were available within Business Central with a focus on the the assets that are deprecated in Version 9.0 too. 
+This is to indicate how to manage assets in absence of Business Central.
+
+.Business Central Asset Types
+[cols="2,1,4,3,3"]
+|===
+| Asset Name | Asset Type | Description | Version 9.0 Supported Tools | Additional Info.
+
+| Business Process | Process (.bpmn) | Modeled as Business Process & Notation (BPMN) diagrams | {DEV_TOOLS} +
+{CANVAS} | Also used for straight-through-processing (STP) as well as rule-flow
+| Decision Model | Decision (.dmn) | Modeled as Decision Modeling & Notation (DMN) diagrams | {DEV_TOOLS} +
+{CANVAS} |
+| Technical Rule | Decision (.drl) | Drools Rule Language, used for technical rules | {DEV_TOOLS} |
+| Decision Table (Spreadsheet) | Decision (.xls) | DRL-based decision table, modeled as an Excel spreadsheet | {DEV_TOOLS} +
+MS Excel Online |
+| Test Scenario | Decision (.scesim) | Test scenario used for decision unit testing | {DEV_TOOLS} |
+| Data Object | Model (.java) | Java data object used by rules and processes | {DEV_TOOLS} | Define as separate Maven dependency project
+| Global Variable(s) | Decision (.drl) | Vocabulary definition | {DEV_TOOLS} | Managed directly in .drl files
+| Domain Specific Language Definition | Decision (.dsl) | Vocabulary definition | **Deprecated** |
+| Enumeration | Model | Vocabulary definition | {DEV_TOOLS} | Managed directly in .drl files
+| Form | Forms (.frm) | Human Form | **Deprecated** | New form builder in future release
+| Guided Decision Table | Decision (.gdst) | Guided Decisison Table | **Deprecated** | **Export** from BC to Decision Table Spreadsheet (XLS)
+| Guided Decision Table Graph | Decision (.gdst-set) | Guided Decision Table Graph| **Deprecated** | 
+| Guided Rule | Decision (.rdrl) | Guided Technical Rule | {DEV_TOOLS} | **Copy** source from BC to new Technical Rule (.drl) file
+| Guided Rule Template | Decision (.template) | Template for guided rules | **Deprecated** |
+| Test Scenario (Legacy)| Decision (.scenario) | Legacy (guided) test scenario used for rule unit testing | **Deprecated** |
+
+|===
+
+
+[NOTE]
+====
+At this time `Test Scenarios` are not directly supported in {CANVAS}, as it has its own test scenario asset.  There will be comptability between test scenarios 
+included in a future release of {PRODUCT}.
```

Comment:
I would mention that Test Scenario is supported in DEV_TOOLS, but not at 100% of its capabilities. DMN-based Test scenario files are fully supported, RULE-based Test Scenario files are not.

Reply:
@yesamer see my proposal https://github.com/IBM/bamoe-docs/pull/46/files/05ac33642e6be1915977c2ba61d454ccccb7313b#r1364826554

Reply:
updated based on jomark's recommendation